packages/internal-build-utils/src/generate-third-party-notice.ts

```diff
@@ -98,6 +98,10 @@ async function getPackageRoot(filename: string): Promise<string | undefined> {
   try {
     const pkgPath = join(dir, "package.json");
     await stat(pkgPath);
+    const pkg = JSON.parse(await readFile(pkgPath, "utf-8"));
+    if (!pkg.name) {
```

Comment:
it's because the tool would crash when the package.json found is "incomplete" because it will try to extract many information from it. I added the check/skip because I think the package should at least have a name if we are expected to get license information from it. But feel free to let me know if there are other better way to do the check.

In detail, I added the change because it crashees when I installed 'vscode-languageserver-textdocument' package which contains a package.json as below...
![image](https://github.com/user-attachments/assets/12a35d1a-71a4-4432-9d2f-6f3b659d0e5f)

Reply:
it's not needed anymore. removed.